### PR TITLE
Update Seven Languages to Xtext 2.11

### PR DESCRIPTION
**Pull request by cdietrich**
Update Seven Languages to Xtext 2.11

- switch to Xtext 2.11
- regenerated Xtext / Xtend
- fixed Xtext Tests
- Moved to Java 8

### CODE REVIEW

**Comment by dhuebner** [comment]
Can one of the admins verify this patch?

**Comment by dhuebner** [comment]
I was about to do the same xD

File: examples/org.xtext.builddsl.examples/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -27,7 +27,7 @@ org.eclipse.jdt.core.compiler.annotation.nullanalysis=disabled
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
 org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (dhuebner) [inline examples/org.xtext.builddsl.examples/.settings/org.eclipse.jdt.core.prefs]: There are still some 1.7 entries. We could fix them as well

File: languages/org.xtext.builddsl/src-gen/org/xtext/builddsl/build/impl/BuildPackageImpl.java [hunk 2]
```diff
@@ -108,6 +108,7 @@ public static BuildPackage init()
     isInited = true;
 
     // Initialize simple dependencies
+    TypesPackage.eINSTANCE.eClass();
```
Comment (dhuebner) [inline languages/org.xtext.builddsl/src-gen/org/xtext/builddsl/build/impl/BuildPackageImpl.java]: Is that new in latest SNAPSHOT? I do not have this line after re-generating

Comment (dhuebner) [inline languages/org.xtext.builddsl/src-gen/org/xtext/builddsl/build/impl/BuildPackageImpl.java]: @cdietrich We should also remove examples/org.xtext.template.examples/xtend-gen 
I think it's old and was moved to .xtend-gen

**Comment by cdietrich** [comment]
ping

**Comment by svenefftinge** [comment]
What about Dennis' feedback? Otherwise +1

**Comment by cdietrich** [comment]
hmm i'd like to do structual cleanups in a second pull request

**Comment by svenefftinge** [comment]
Sure, go ahead

**Comment by cdietrich** [comment]
implemented dennis feedback. i think we can merge now. can we?

**Comment by cdietrich** [comment]
ping

**Comment by svenefftinge** [comment]
Yes, please merge.

**Comment by cdietrich** [comment]
i cant :( see some comments above

**Pull request merged**